Source/Scene/PointCloud3DTileContent.js

```diff
@@ -1060,17 +1070,28 @@ define([
 
         vs += '} \n';
 
-        var fs = 'varying vec4 v_color; \n' +
-                 'uniform bool u_clippingPlanesEnabled; \n' +
-                 'uniform int u_clippingPlanesLength;' +
-                 'uniform vec4 u_clippingPlanes[czm_maxClippingPlanes]; \n' +
-                 'void main() \n' +
-                 '{ \n' +
-                 '    if (u_clippingPlanesEnabled) { \n' +
-                 '        czm_discardIfClipped(u_clippingPlanes, u_clippingPlanesLength); \n' +
-                 '    } \n' +
-                 '    gl_FragColor = v_color; \n' +
-                 '} \n';
+        var fs = 'varying vec4 v_color; \n';
+
+        if (hasClippingPlanes) {
+            fs += 'uniform int u_clippingPlanesLength;' +
+                  'uniform vec4 u_clippingPlanes[czm_maxClippingPlanes]; \n' +
+                  'uniform vec4 u_clippingPlanesEdgeColor; \n' +
+                  'uniform float u_clippingPlanesEdgeWidth; \n';
+        }
+
+        fs +=  'void main() \n' +
+               '{ \n' +
+               '    gl_FragColor = v_color; \n';
+
+        if (hasClippingPlanes) {
+            var clippingFunction = clippingPlanes.combineClippingRegions ? 'czm_discardIfClippedCombineRegions' : 'czm_discardIfClipped';
```

Comment:
Ideally this should go in the vertex shader so we can clip points before they go to the frag shader, but this is ok for now considering these functions call `discard`.